be/src/column/object_column.h

```diff
@@ -168,11 +168,21 @@ class ObjectColumn : public ColumnFactory<Column, ObjectColumn<T>> {
         return _cache;
     }
 
+    Buffer<T*>& get_proxy_data() {
+        _build_cache();
+        return _cache;
+    }
+
     const Buffer<T*>& get_data() const {
```

Comment:
```
A common best practice is to avoid code duplication. The code within the get_proxy_data() function and get_data() function is identical, which seems unnecessary. Try consolidating these methods if they're supposed to do the same work.
```

Reply:
Fixed